### PR TITLE
Clarify filtering of audit messages and individual properties

### PR DESCRIPTION
**Pull request by andreasohlund**
Clarifying audit filtering after a chat with @SimonCropp

### CODE REVIEW

File: nservicebus/operations/auditing.md [hunk 1]
```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
 ## Configuring auditing
 
 partial: configuration
+
+## Filtering audit messages
+
+By default all messages are audited, to control what message types that gets audited see our [audit filter sample](/nservicebus/audit-filter/).
```
Comment (timbussmann) [inline nservicebus/operations/auditing.md]: could use some little tweaks. How about:
By default all messages are audited. To control which message types are audit, see the [audit filter sample]

File: nservicebus/operations/auditing.md [hunk 2]
```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
...
+
+### Filtering out individual properties
```
Comment (timbussmann) [inline nservicebus/operations/auditing.md]: Filtering individual properties

File: nservicebus/operations/auditing.md [hunk 3]
```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
...
+
+Auditing works by sending an exact copy of the received message to the audit queue so filtering out individual propeties is not supported.
```
Comment (timbussmann) [inline nservicebus/operations/auditing.md]: spelling mistake in "properties"

Comment (timbussmann) [inline nservicebus/operations/auditing.md]: should there be a comma between "[...] to the audit queue, so filtering out [...]"?

File: nservicebus/operations/auditing.md [hunk 4]
```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
...
+
+Property filtering requirements usually comes from properties either containing sensitive information like credit card numbers and passwords or them being large in size like videos and images.
+
+For sensitive properties we recommend property [encryption](/nservicebus/security/property-encryption/) and for large properties we recommend to [stream them instead](/samples/pipeline/stream-properties/) to avoid including the actual payload in the audited message.
```
Comment (timbussmann) [inline nservicebus/operations/auditing.md]: If you change the sentence to something like this:
For sensitive properties (e.g. credit card numbers or password), we recommend property encryption [...].
you could remove the entire section above.

the encryption link should also include the word property, as the feature is called "message property encryption".

Comment (bording) [reply nservicebus/operations/auditing.md]: Also needs a comma "For sensitive properties,"

File: nservicebus/operations/auditing.md (hunk 1 above)
Comment (SimonCropp) [reply nservicebus/operations/auditing.md]: That link does not point to a sample

**Pull request merged**